packages/client/modules/meeting/components/ScaleDropdownMenuItem.tsx

```diff
@@ -61,42 +63,50 @@ const ScaleActionButtonGroup = styled('div')({
   marginBottom: 'auto'
 })
 
-const ScaleDropdownMenuItem = forwardRef((props: Props, ref) => {
-  const {scale, dimension, menuProps} = props
-  const {closePortal} = menuProps
-  const {values} = scale
+const StarterIcon = styled(Icon)({
+  fontSize: 18,
+  lineHeight: '24px',
+  paddingLeft: 4
+})
 
+const ScaleDropdownMenuItem = forwardRef((props: Props, ref) => {
+  const {scale, dimension, closePortal, scaleCount} = props
+  const {id: scaleId, isStarter, name: scaleName, values} = scale
+  const {id: dimensionId, team} = dimension
+  const {id: teamId} = team
   const atmosphere = useAtmosphere()
   const {submitMutation, submitting, onError, onCompleted} = useMutationProps()
 
   const setScale = (scaleId: any) => () => {
     if (submitting) return
     submitMutation()
-    UpdatePokerTemplateDimensionScaleMutation(atmosphere, {dimensionId: dimension.id, scaleId}, {onError, onCompleted})
+    UpdatePokerTemplateDimensionScaleMutation(atmosphere, {dimensionId, scaleId}, {onError, onCompleted})
     closePortal()
   }
 
   return (
     <MenuItem
       ref={ref}
-      key={scale.id}
```

Comment:
key is only needed if `map` is called in the same component. you already passed a key to this component in the parent, so this is good to go